File: 2108-find-first-palindromic-string-in-the-array/2108-find-first-palindromic-string-in-the-array.java
class Solution {
    public String firstPalindrome(String[] words) {
        for(int i = 0; i < words.length; i++){
            if(checkPalindrom(words[i])){
                return words[i];
            }
        }
        return "";
    }

    private boolean checkPalindrom(String str){
        String reversedStr = new StringBuilder(str).reverse().toString();
        if(str.equals(reversedStr)){
            return true;
        }
        return false;
    }
}

File: 2108-find-first-palindromic-string-in-the-array/2108-find-first-palindromic-string-in-the-array.js
/**
 * @param {string[]} words
 * @return {string}
 */
var firstPalindrome = function (words) {
    for (let i = 0; i < words.length; i++) {
        if (reverseString(words[i])) {
            return words[i];
        }
    }
    return "";
};

function reverseString(str) {
    reversedStr = str.split("").reverse().join("");
    if (str === reversedStr) {
        return true;
    }
    return false;
}
